client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-credentials-form.ts

```diff
@@ -1,13 +1,19 @@
-import { useEffect } from 'react';
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
-import { useFormErrorMapping } from './hooks/use-form-error-mapping';
-import { CredentialsFormData } from './types';
+import { CredentialsFormData } from '../types';
+import { useFormErrorMapping } from './use-form-error-mapping';
 import { useSiteMigrationCredentialsMutation } from './use-site-migration-credentials-mutation';
 
 export const useCredentialsForm = ( onSubmit: () => void ) => {
 	const importSiteQueryParam = useQuery().get( 'from' ) || '';
+	const [ requestInAnywayMode, setRequestInAnywayMode ] = useState( false );
```

Comment:
Maybe this could use a more descriptive name? I had to track down the logic to know what it does. WDYT of `canBypassVerification`?

Reply:
Thanks for the suggestion! Updated it here https://github.com/Automattic/wp-calypso/pull/94869/commits/fdb25e3edea2ec423d3664bee8d280f7c64017d5